eo-maven-plugin/src/test/java/org/eolang/maven/it/SnippetTestCase.java

```diff
@@ -301,18 +303,23 @@ private static String jdkExecutable(final String name) {
      * @return Classpath.
      */
     private static String classpath() {
-        return String.format(
-            ".%s%s",
-            File.pathSeparatorChar,
-            System.getProperty(
-                "runtime.jar",
-                Paths.get(System.getProperty("user.home")).resolve(
+        final String runtime;
+        final String property = System.getProperty("runtime.jar");
+        if (Objects.isNull(property) || property.isEmpty() || StringUtils.isBlank(property)) {
```

Comment:
@volodya-lombrozo isn't `StringUtils.isEmpty` not enough? https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isEmpty-java.lang.CharSequence-

Reply:
@volodya-lombrozo I would say, in this case we don't care. We just want to say "if it's not a real path, ..." An empty path/string can't be a real path, right?

Reply:
> @volodya-lombrozo I would say, in this case we don't care. We just want to say "if it's not a real path, ..." An empty path/string can't be a real path, right?

@yegor256 yes, you are right.